assets/tests/components/mapPage.test.tsx

```diff
@@ -322,7 +318,7 @@ describe("<MapPage />", () => {
       name: /vehicle properties card/i,
     })
 
-    expect(mapSearchPanel).not.toBeVisible()
+    expect(mapSearchPanel).toHaveClass("hidden")
```

Comment:
suggestion: I think you should revert this and change the `const mapSearchPanel` to be `screen.getByRole("form", { name: /search map/i })` or something of the like

And then make the form in the mapPage conform to the test.

Reply:
(trying to come up with an example for you)

Reply:
Are you thinking of using `aria-hidden` or something? In any event now that I think about it we should add some ARIA label or other to indicate that the panel is hidden away from view when it's collapsed.

Reply:
I can't seem to highlight the contents in mapPage that I think should be changed, but when you hide the search panel there's still a scroll bar (and apparently the firefox screenshot tool can screenshot things outside of the view port)
![image](https://user-images.githubusercontent.com/6913198/213463467-d9c86b3e-4bb3-4713-9686-f203c8768ef4.png)

What do you think of the following? 
[wrap the form and search results in a `div.m-map-page__input-and-results`(moving the class from the parent div, and do `display: none` on that `div` when in the hidden state, and have the drawer tab be outside of that ](https://github.com/mbta/skate/pull/1889)

Reply:
> Are you thinking of using `aria-hidden` or something? In any event now that I think about it we should add some ARIA label or other to indicate that the panel is hidden away from view when it's collapsed.

`display: none` on any of the parents of the form should result in `.not.toBeVisible()` to succeed, so I don't think we need aria-hidden, and if we did, I think we should use `hidden` first. I've considered suggesting styling the hidden state, but I'm a bit concerned that even though that's allowed, it might break other accessibility tools with different expectations and handling.

Reply:
Sure that works, could you create the tech debt ticket and link this comment thread so that the test is changed to use `not.toBeVisible()` again?
or add it as an AC, that works too.

Reply:
Will do! For what it's worth I think the approach in your branch looks like a good one, it's just a matter of ironing out any wrinkles.